### DISCUSSION TITLE
Do we want to include knowledge graph related functions in the iteration 1 prototype

### DISCUSSION THREAD

**Discussion by roblokers** [soilwise-he/SoilWise-documentation#30] [Ideas]
This was raised in the 02/05/24 WP3 meeting.
It would allow stakeholders to get a 1st glimps of the possibilities early in the project. However, there's still quite some foundational work to be done and integrating this into the current setup for the Sept. deliverable might be difficult.

**Comment by BerkvensNick** [comment]
In the 10 functionalities we have from JRC, nr5 is: 
"The SWR knowledge graph will also be enhanced by links to knowledge from external resources, starting with CORDIS. To a user, the knowledge from the knowledge graph will be available, among others, through interlinked metadata, i.e., metadata linked to relevant deliverables of various projects, etc. "
So I think we should try to have a knowledge graph as part of the infrastructure for the Sept. deliverable

**Comment by pvgenuchten** [comment]
KG is available (but not harmonised yet)
